### ISSUE TITLE
How to change settings when installed via pip?

### ISSUE THREAD

**Issue by EliasPereirah** [VikParuchuri/marker#162]

**Comment by huangh12** [comment]
I am also puzzled by this..

**Comment by gmotta** [comment]
A command similar to:
pip show marker-pdf 

Should give you what you are looking for:

In my case:
<user>@<machine> markerexperiments % python3 -m pip show marker-pdf
Name: marker-pdf
Version: 0.2.6
Summary: Convert PDF to markdown with high speed and accuracy.
Home-page: https://github.com/VikParuchuri/marker
Author: Vik Paruchuri
Author-email: github@vikas.sh
License: GPL-3.0-or-later
Location: /Users/<user>/.pyenv/versions/3.9.19/lib/python3.9/site-packages
Requires: filetype, ftfy, grpcio, numpy, pdftext, Pillow, pydantic, pydantic-settings, python-dotenv, rapidfuzz, ray, regex, scikit-learn, surya-ocr, tabulate, texify, torch, tqdm, transformers
Required-by:

**Comment by Matthijz98** [comment]
You can look at https://github.com/VikParuchuri/marker/blob/master/marker/settings.py for what the default settings are.
Then u can overwrite one by setting a env (google that for your platform)
The text in the README.md could make that a bit more clear and support for passing all options as arguments would be nice

**Comment by smortezah** [comment]
It would be nice to have a TOML config file, e.g. `config.toml`, instead of changing the settings.py file.
